files/ru/web/xpath/functions/floor/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Floor
+slug: Web/XPath/Functions/floor
+translation_of: Web/XPath/Functions/floor
+original_slug: Web/XPath/Funkcje/floor
+---
+{{ XsltRef() }}
+
+Функция `floor` принимает десятичное число и возвращает наибольшее целое, меньшее или равное исходному.
+
+### Синтаксис
+
+```
+floor( number )
+```
+
+### Аргументы
+
+- `number`
+  - : Десятичное число
+
+### Возвращаемое значение
+
+Ближайшее целое, меньшее или равное `number`.
+
+### Определено
+
+[XPath 1.0 4.4](http://www.w3.org/TR/xpath#function-floor)
+
+### Поддержка Gecko
+
+Поддерживается
+
+{{ languages( { "fr": "fr/XPath/Fonctions/floor", "ja": "ja/XPath/Functions/floor", "pl": "pl/XPath/Funkcje/floor", "en": "en/XPath/Fonctions/floor" } ) }}
```

Comment:
Do we need this macro?

Reply:
No we don't, `{{languages}}` can be removed (it's a source of possible later errors due to https://github.com/mdn/yari/issues/4744 and https://github.com/mdn/translated-content/issues/5603

Reply:
I'd say such a removal may has its place in a "cleanup" PR. Do you want me to open a PR to remove `{{languages}}` for this section and rebase this current PR on top of it?